upgrade.go

```diff
@@ -675,6 +685,57 @@ func (b *providerUpgradeBuilder) providerUpgradeRecordBaselines(t *testing.T) {
 	integration.ProgramTest(t, &test)
 }
 
+func SanitizeSecretsInStackState(stack *apitype.UntypedDeployment) (*apitype.UntypedDeployment, error) {
```

Comment:
This seems to be duplicated from `sanitize.go`, but without the tests. Could we re-use the functions between the two usages?

Reply:
yep, wanted to remove this and forgot